package.json

```
@@ -28,6 +28,8 @@
     "desktop:serve:quicklook": "webpack serve --mode development --progress --config desktop/webpack.quicklook.config.ts",
     "desktop:start": "electron desktop/.webpack",
     "web:build:dev": "webpack --mode development --progress --config web/webpack.config.ts",
+    "web:build:dev:fast": "webpack --mode development --progress --config web/webpack.config.optimized.ts",
```

Biome (2.1.2)

[error] The key web:build:dev:fast was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)